log4j-java-maven/src/main/java/com/log4j2/Error.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache license, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the license for the specific language governing permissions and
+ * limitations under the license.
+ */
+package com.log4j2;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Error {
```

Comment:
What is the point of this class?
FTR: No Javadoc. Wrong package anyway.
It feels like there is a disconnect between what should be an example and your expectations. -1 unless someone can explain why we need this class or this PR.

Reply:
OK, the package name is just an example name (I just make a demo name for it ).
I have made 2 classes (Main and Error class)
Both classes are used for logging. 
The Main class is responsible for Debug logging and the Error Class is mainly used for error logging.
If you think it's confusing I can rename the package and class name for a better understanding. Thanks

Reply:
++ Updated the package and Class name.